src/components/skeleton/Drawer.vue

```diff
@@ -0,0 +1,187 @@
+
+        <v-list-item-content>
+          <v-list-item-title> Mail </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+       <v-list-item link :href="githubURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-github</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> Github </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+       <v-list-item link :href="twitterURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-twitter</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> Twitter </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+       <v-list-item link :href="youtubeURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-youtube</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> Youtube </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+    </v-list>
+
+    <v-spacer></v-spacer>
+
+    <v-list-group
+      no-action
+      append-icon="mdi-chevron-down"
+      prepend-icon="mdi-translate"
+    >
+      <template #activator>
+        <v-list-item-title
+          class="body-2"
+          v-text="$t('appbar.language')"
+        />
+      </template>
+
+      <v-list-item
+        @click="$i18n.locale = 'fr'"
+      >
+        <v-list-item-title class="body-2" v-text="$t('appbar.french')"/>
+        <v-list-item-icon>
+          <img width="24" :src="require('../../../public/images/lang/fr.png')" alt="fr">
+        </v-list-item-icon>
+      </v-list-item>
+
+      <v-list-item
+        @click="$i18n.locale = 'en'"
+      >
+        <v-list-item-title class="body-2" v-text="$t('appbar.english')"/>
+        <v-list-item-icon>
+          <img width="24" :src="require('../../../public/images/lang/en.png')" alt="en">
+        </v-list-item-icon>
+      </v-list-item>
+    </v-list-group>
+  </v-navigation-drawer>
+</template>
+
+<script>
+export default {
+  name: 'Drawer',
+  data() {
+    return {
+      appVersion: '1.0.0',
+
+      blogURL: 'https://blog.ezpaarse.org/',
+      analogistURL: 'http://analyses.ezpaarse.org/',
+      ezpaarseDemoURL: 'http://demo.ezpaarse.org/',
+      ezmesureURL: 'https://ezmesure.couperin.org/',
+      bibliomapURL: 'https://bibliomap.inist.fr/',
+      ezpaarseDocURL: 'https://ezpaarse-project.github.io/ezpaarse/',
+
+      mailURL: 'mailto:ezteam@couperin.org',
+      githubURL: 'https://github.com/ezpaarse-project',
+      twitterURL: 'https://twitter.com/ezpaarse',
+      youtubeURL: 'https://www.youtube.com/channel/UCcR-0UE9WjYiwS4fMG2T4tQ',
+    };
+  },
+  computed: {
+    drawer: {
+      get() {
+        if (this.$vuetify.breakpoint.lgAndUp) {
+          return false;
+        }
+        return this.$store.state.drawer;
+      },
+      set(newVal) {
+        this.$store.dispatch('setDrawer', newVal);
+      },
+    },
```

Comment:
A two way binding (cf [v-model](https://v2.vuejs.org/v2/guide/components.html#Using-v-model-on-Components) or [.sync](https://v2.vuejs.org/v2/guide/components-custom-events.html#sync-Modifier)) will be less heavy than a whole store for just a menu that `App.vue` can handle + it's totally overkill...